### PR TITLE
#561 - Overall Frontend Refactor (excludes Exercise)

### LINKED ISSUE #561: Story: Refactor Overall Styling (Frontend)
# Refactor Overall Styling

As a **user**, I want **a better user experience**, so that **I will enjoy my interaction with the website more**.

Associated Epic: Lab 0

## Definition of Done

- [ ] Acceptance criteria defined
- [ ] Solution tasks specified
- [ ] Feature branch created
- [ ] ~~Unit tests written (on API code only for now)~~
- [ ] ~~Solution passes full suite of unit tests~~
- [ ] Design documents updated if needed
- [ ] Pull request created
- [ ] Code is reviewed
- [ ] Code is merged to `develop` branch
- [ ] ~~Tests pass on CI~~
- [ ] ~~Deployed to staging environment~~
- [ ] Acceptance criteria is met

## Acceptance Criteria

- [ ] Given **I navigate to ALL**, when **i go to the any lab page**, then **a newly designed header will appear**
- [ ] Given **I navigate to ALL**, when **I go to the About page**, then **it will have new styling**
- [ ] Given **I navigate to ALL**, when **I go to the Reinforcement Page**, then **it will have new styling**
- [ ] Given **I navigate to ALL**, when **I go to the Quiz Page**, then **it will have new styling**
- [ ] Given **I navigate to ALL**, when **I reach the landing page**, then **the Navbar ALL logo has been updated as well as text styling**

## Solution Tasks

- [ ] Update Body Header
- [ ] Update Header/Navbar
- [ ] Update About Page
- [ ] Update Reinforcement Page
- [ ] Update Quiz Page

### PR DESCRIPTION
**Pull request by dm9965**
- [ ] No discrepancies across browsers. Eg. chrome vs safari
- [ ] Pages pass W3C Validation for HTML (https://validator.w3.org)
- [ ] CSS (https://jigsaw.w3.org/css-validator/)
- [ ] Google sign-in is operational
- [ ] Accessibility functions
- [ ] Pages can scale without distorting page
- [ ] No dead links
- [ ] Navbar is consistent across the site
- [ ] Pages are screen-reader accessible
- [ ] Contrast meets standards for accessibility
- [ ] Pages are keyboard accessible
- [ ] Passes WAVE Evaluation Tool chrome extension

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -8,6 +8,16 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
```
Comment (heathermoses) [inline client/public/index.html:20]: Nit: google fonts get imported in the `main.css` file. We can go back and change this later

File: client/src/assets/images/logos/FinalALLLogo.png
Comment (heathermoses) [inline client/src/assets/images/logos/FinalALLLogo.png:1]: Nit: looks like there's two logo files, do we need both?

**Review by heathermoses — APPROVED** [review]
Small nitpicks, overall good!

**Comment by Mster115** [comment]
nit: 
image
image
whole header spanning entire page while the rest of the body is stuck together doesn't flow right to me feels stretched

**Comment by Mster115** [comment]
favicon needs to be updated with new logo. ie. one without text
nit: the header color feels out of place, idk what color would make it better. The header color seems like this is the only spot where the color is used. the footer on the homepage is also a different color so it feels off. maybe fix the footer image on main page? maybe the nav buttons on the bottom could be the same color as the nav header? again this is a nit pick
image
image


Overall it looks amazing!

**Pull request merged**